runtime/lua/vim/_init_packages.lua

```diff
@@ -50,7 +50,25 @@ table.insert(package.loaders, 2, vim._load_package)
 
 -- builtin functions which always should be available
 require'vim.shared'
-vim.inspect = require'vim.inspect'
+
+vim._submodules = {inspect=true}
+
+-- These are for loading runtime modules lazily 
```

Comment:
```suggestion
-- These are for loading runtime modules lazily
```
(to shut up lualint)